src/main/java/org/broadinstitute/consent/http/resources/UserResource.java

```diff
@@ -227,17 +227,15 @@ private boolean canUpdateInstitution(User user, Integer newInstitutionId) {
             if (user.hasUserRole(UserRoles.ADMIN)) {
                 return true; // admins can do everything.
             }
-
             if (user.hasUserRole(UserRoles.SIGNINGOFFICIAL) || user.hasUserRole(UserRoles.ITDIRECTOR)) {
                 // can only update institution if not set.
                 return Objects.isNull(user.getInstitutionId()) && Objects.nonNull(newInstitutionId);
             }
-
-            return false;
+            // User is not restricted based on role
+            return true;
```

Comment:
should we add a quick test for this?

Reply:
Yeah, I should have done that. The method is private and only used once - will open it up for tests